discovery-infra/test_infra/helper_classes/config/controller_config.py

```diff
@@ -5,11 +5,8 @@
 from dataclasses import dataclass
 
 from test_infra import consts
-from test_infra.utils.global_variables import GlobalVariables
 from .base_config import _BaseConfig
 
-global_variables = GlobalVariables()
```

Comment:
Where is it being assigned instead?

Reply:
Where it was initiated from the first place. In `global_configs.py`